e2e/support/test_support.go

```diff
@@ -1719,9 +1719,9 @@ func UserCleanup() {
 		command.Stderr = os.Stderr
 		command.Stdout = os.Stdout
 		if err := command.Run(); err != nil {
-			fmt.Printf("An error occurred during user cleanup command execution: %v\n", err)
+			fmt.Fprintf(command.Stderr, "An error occurred during user cleanup command execution: %v\n", err)
```

Comment:
Could be replaced with `Fprintln`.

Reply:
Ok. I'd make the change

Reply:
Looking at this again, `Fprintln` doesn't accept a format string. Using `Fprintln` would mean doing away with the formatting. Isn't that inportant?

Reply:
`Fprintln` operands are appended with white spaces, e.g.:

```golang
fmt.Fprintf(command.Stderr, "An error occurred during user cleanup command execution: %v\n", err)
```

Can be turned into:

```golang
fmt.Fprintln(command.Stderr, "An error occurred during user cleanup command execution:", err.Error())
```

I think for all the statements where formatting is actually used for appending strings, then `Fprintln` is more appropriate.

Reply:
Ok thank you. I'll use that